worker.proto

```diff
@@ -0,0 +1,78 @@
+// proto file to define the request / response messages and gRPC service for job worker
+syntax = "proto3";
+
+package JobWorker;
+
+option go_package = "github.com/bkneis/jobworker";
+option java_multiple_files = true;
+option java_package = "com.teleport.jobworker";
+option java_outer_classname = "JobWorkerProto";
+
+// Start request containing linux command as a string and resource control options
+message StartRequest {
+    string name = 1;
+    repeated string args = 2;
+    JobOpts opts = 3;
+}
+
+// All other requests just have job UUID. I did not make these generic as per protobuf best practices
+message StopRequest {
+    string id = 1;
+}
+
+message StatusRequest {
+    string id = 1;
+}
+
+message OutputRequest {
+    string id = 1;
+}
+
+// Options for cgroup v2 controllers, see doc.go for example interfaces
+message JobOpts {
+    int32 cpu_weight = 1;
+	string mem_limit = 2;
```

Comment:
Allowing these values to be specified as strings in user-facing UX (e.g., CLI flags) makes sense, but I’m on the fence about whether it’s the best interface for the gRPC API / Go library. Curious how about your reasoning.

Reply:
Originally I used strings since the memory and IO controller have a unit of measure in the value, i.e. "M" or "g" for memory and "ms" for IO. Looking at it now IO only has 1 unit so this can be changed to an int. To keep it consistent with the other parameters I have also changed memory to be an int where "M" is assumed as the unit. This will also be documented in the CLI help function and library.

Reply:
Instead of assuming mega-units, could we accept the actual cgroup value? Then you could provide consts in Go to make the API easier to use (similar to `time.Second`).